src/expand.rs

```diff
@@ -265,6 +265,14 @@ fn run_tests<I, S>(
     }
 }
 
+fn matches_expected(expected: &str, actual: &str) -> bool {
```

Comment:
Please add doc comments. 
I know the code base doesn't have too much of it itself, but I'd like new code to improve on it.